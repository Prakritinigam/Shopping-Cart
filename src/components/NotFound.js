import withHOC from "./withHOC";

function NotFound() {
    return <h2>Oops, Please check if you have typed the correct link.</h2>
}

export default withHOC(NotFound);
