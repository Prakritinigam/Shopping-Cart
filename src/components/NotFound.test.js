import { render } from '@testing-library/react';
import NotFound from './NotFound';
import React from 'react';

it('renders', () => {
  const { asFragment } = render(<NotFound/>);
  expect(asFragment()).toMatchSnapshot();
});