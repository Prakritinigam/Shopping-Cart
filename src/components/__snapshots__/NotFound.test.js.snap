// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div
    style="text-align: center;"
  >
    <h2>
      Oops, Please see you have typed the correct link.
    </h2>
    <a
      href="/"
    >
      Shopping Cart
    </a>
  </div>
</DocumentFragment>
`;
